matchmake-extension/auto_matchmake_with_param_postpone.go

```diff
@@ -37,10 +63,19 @@ func (commonProtocol *CommonProtocol) autoMatchmakeWithParamPostpone(err error,
 		session = sessions[0]
 	}
 
-	errCode := common_globals.AddPlayersToSession(session, []uint32{connection.ID}, connection, "")
-	if errCode != nil {
-		common_globals.Logger.Error(errCode.Error())
-		return nil, errCode
+	if useAdditionalParticipants {
+		// * Move everyone. They're still connected to their old session
+		errCode := common_globals.MovePlayersToSession(session, additionalParticipants, connection, "")
```

Comment:
The last parameter should be the join message, which is present inside `AutoMatchmakeParam`

same for adding players I believe